### DISCUSSION TITLE
Importing Models from NPM Package

### DISCUSSION THREAD

**Discussion by enszrlu** [vladmandic/human#416] [Q&A] [answered]
Maybe this is too basic and silly but I am trying to use this library with node. As models are coming within npm package (@vladmandic/human/models), I expected Human() function to use these as default, but I believe default path to models is ./models instead of ./node_modules/@vladmandic/human/models. 

What is the right way of importing models from node package? Below config works but I am not sure if this is right way of doing it. 

``` const humanConfig = {
    // backend: 'tensorflow',
    **modelBasePath: "file://node_modules/@vladmandic/human/models",**
    debug: true,
    async: false,
    filter: {
      enabled: true,
      flip: true,
    },
    face: {
      enabled: true,
      detector: { enabled: true, rotation: false },
      mesh: { enabled: true },
      iris: { enabled: true },
      description: { enabled: true },
      emotion: { enabled: true },
    },
    hand: {
      enabled: true,
    },
    // body: { modelPath: 'blazepose.json', enabled: true },
    body: { enabled: true },
    object: { enabled: true },
  }; ```

**Comment by vladmandic** [comment, marked as answer]
you're correct.
and there is no single best default that works for all.
if you use `human` as a library for nodejs, its as you've stated. but if you use it as a library in browser, its different and if you use it in nodejs as a clone, its different again.

**Reply by enszrlu** [reply]
Wow, it was quickest response I have ever get. So my model path is how it is suppose to be?

**Reply by vladmandic** [reply]
looks good to me
